Lib/idlelib/config-main.def

```diff
@@ -67,6 +67,7 @@ font-size= 10
 font-bold= 0
 encoding= none
 line-numbers-default= 0
+strip-trailing-whitespace-on-save= 1
```

Comment:
`rstrip-on-save` might be sufficient.

Except for line-numbers, the other existing items are actually for the base class used for editor, shell, and output windows.  This has to be for editor windows only, at least at present.  PyShell subclasses code.Interactive Interpreter, which already does some stripping before compiling.  And user code output should not be touched.  This different between items should somehow be more obvious on the configdialog tab.  I am thinking about what to do.

If someone want to disable this, doing so for particular windows makes more sense than globally.  Could we get away with having only a window option, or are there really people who would never want